package.json

```diff
@@ -1,60 +1,55 @@
-    "@emotion/react": "^11",
-    "@emotion/styled": "^11",
-    "@growthbook/growthbook": "^0.18.0",
-    "@testing-library/jest-dom": "^5.16.1",
-    "@testing-library/react": "^12.1.2",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.4.0",
-    "@types/node": "^16.11.18",
-    "@types/react": "^17.0.38",
-    "@types/react-dom": "^17.0.11",
-    "framer-motion": "^5",
-    "json-stringify-pretty-compact": "^3.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-hook-form": "^7.22.5",
-    "react-icons": "^4.3.1",
-    "react-scripts": "5.0.0",
-    "react-syntax-highlighter": "^15.4.5",
-    "typescript": "^4.5.4",
-    "url": "^0.11.0",
-    "web-vitals": "^2.1.2"
-  },
   "scripts": {
-    "start": "BROWSER=none react-scripts start",
-    "build": "yarn clean && GENERATE_SOURCEMAP=false INLINE_RUNTIME_CHUNK=false craco build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "clean": "rm -rf build",
-    "package": "yarn build && cd build && zip -r build.zip . && mv build.zip ../"
+    "dev": "webpack --config webpack/webpack.dev.js --watch",
+    "build": "webpack --config webpack/webpack.prod.js",
+    "clean": "rimraf dist",
+    "test": "npx jest",
+    "style": "prettier --write \"src/**/*.{ts,tsx}\""
   },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/growthbook/devtools"
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "dependencies": {
+    "@chakra-ui/react": "^2.4.9",
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@growthbook/growthbook": "^0.20.1",
+    "clsx": "^1.2.1",
+    "framer-motion": "^8.4.3",
+    "json-stringify-pretty-compact": "^4.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-hook-form": "^7.42.1",
+    "react-icons": "^4.7.1",
+    "react-syntax-highlighter": "^15.5.0"
   },
   "devDependencies": {
-    "@types/chrome": "^0.0.174",
-    "@types/react-syntax-highlighter": "^13.5.2"
+    "@svgr/webpack": "^6.5.1",
+    "@types/chrome": "0.0.158",
+    "@types/jest": "^27.0.2",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "@types/react-syntax-highlighter": "^15.5.6",
+    "autoprefixer": "^10.4.13",
+    "copy-webpack-plugin": "^9.0.1",
+    "css-loader": "^6.7.3",
+    "glob": "^7.1.6",
+    "jest": "^27.2.1",
+    "postcss": "^8.4.21",
+    "postcss-loader": "^7.0.2",
+    "prettier": "^2.2.1",
+    "rimraf": "^3.0.2 ",
+    "style-loader": "^3.3.1",
+    "tailwindcss": "^3.2.4",
+    "ts-jest": "^27.0.5",
+    "ts-loader": "^8.0.0",
+    "typescript": "^4.4.3 ",
+    "webpack": "^5.61.0",
+    "webpack-cli": "^4.0.0",
```

Comment:
Looks like the webpack CLI is 1 major version behind the library, wondering if there's going to be problems with that.

Also, did the latest version of webpack (5.75.x) not work? Webpack config can grow to be a bit of a pain, so probably good if we can get on the latest version for all webpack tools, if we can.